spras/analysis/cytoscape.py

```diff
@@ -46,14 +46,12 @@ def run_cytoscape(pathways: List[Union[str, PurePath]], output_file: str, contai
         # Provided the mapped pathway file path and the original file path as the label Cytoscape
         command.extend(['--pathway', f'{mapped_pathway}|{pathway}'])
 
-    print('Running Cytoscape with arguments: {}'.format(' '.join(command)), flush=True)
-
     container_suffix = "py4cytoscape:v3"
-    out = run_container(container_framework,
-                        container_suffix,
-                        command,
-                        volumes,
-                        work_dir,
-                        env)
-    print(out)
+    run_container_and_log('Cytoscape',
```

Comment:
Does this new approach to calling the container helper function change anything in the contributing guide?